github-management-script/branch-protection-ruleset.sh

```diff
@@ -22,7 +22,7 @@ set_branch_protection() {
     DATA='{
         "required_status_checks": {
             "strict": true,
-            "contexts": ["Python package / build"]
+            "contexts": ["lint-test / build"]
```

Comment:
ah, yes, that's the fix I expected

Reply:
I think it's wrong again looking at the PR from finesse-frontend

Reply:
I changed contexts to checks to address this comment
https://github.com/ai-cfia/devops/issues/19#issuecomment-1812649958

Also changed "lint-test / build" to "lint-test / lint-test" reasoning explained there:
https://github.com/ai-cfia/github-workflows/pull/28